### DISCUSSION TITLE
What should I pay attention to when deploying to a production environment?

### DISCUSSION THREAD

**Discussion by djleeyuanfang** [ssh-mitm/ssh-mitm#173] [General]
Is there any security issues

**Comment by manfred-kaiser** [comment]
When deploying SSH-MITM to a production environment, it is crucial to ensure that it is used exclusively for auditing purposes.

SSH-MITM should be used strictly for auditing SSH sessions. It is designed to intercept and analyze SSH traffic, which is invaluable for security audits and troubleshooting, but it should not be used for regular operational purposes like a jump host.

Using SSH-MITM as a jump host is not recommended because the audit functionalities inherently expose SSH session details that could compromise the security of the environment. The focus of SSH-MITM on intercepting and analyzing traffic means it does not provide the security guarantees required for a secure jump host.

Be aware of the inherent risks associated with using any MITM (Man-in-the-Middle) tool. Ensure that only authorized personnel have access to the server running SSH-MITM.
